builder/vmware/vmx/step_clone_vmx.go

```diff
@@ -115,13 +115,13 @@ func (s *StepCloneVMX) Run(_ context.Context, state multistep.StateBag) multiste
 	}
 
 	// Write out the relative, host filesystem paths to the disks
-	var diskFullPaths []string
+	var diskPaths []string
 	for _, diskFilename := range diskFilenames {
 		log.Printf("Found attached disk with filename: %s", diskFilename)
-		diskFullPaths = append(diskFullPaths, filepath.Join(s.OutputDir, diskFilename))
+		diskPaths = append(diskPaths, filepath.Join(s.OutputDir, diskFilename))
 	}
 
-	if len(diskFullPaths) == 0 {
```

Comment:
Is there a reason you changed this variable name, other than avoiding potential namespacing issues?

Reply:
`full_disk_path` sounded like a path to a disk that was full to me, although looking at it, `diskFullPaths` could be read the same way - just the way my brain works I guess. `full_paths_to_disks` would avoid any potential ambiguity, but is a little long...
Also, we are now storing *paths* rather than a *single* path...
I also have a personal preference for using camelcase over underscores as it makes the variable name slightly shorter...